pkg/util/api_test.go

```diff
@@ -86,6 +86,35 @@ var _ = Describe("Utils test", func() {
 		})
 	})
 
+	Describe("IsCLIFriendly", func() {
+		Context("name with characters that are not alaphanumeric, period or hyphens", func() {
+			cases := []string{
+				"name with whitespace", "name=test", "invalid[name", "invalid+name", "invalid&name",
+				"invalid%name", "invalid#name", "invalid$name", "invalid*name", "invalid@name", "invalid!name",
+				"invalid;name", "invalid)name", "invalid?name", "invalid^name", "invalid~name", "invalid`name",
+			}
+			for _, v := range cases {
+				name := v
+				It(fmt.Sprintf("%s should be considered not CLI-friendly", name), func() {
+					Expect(util.IsCLIFriendly(name)).To(Equal(false))
+				})
+			}
+		})
+		Context("name with only alaphanumeric, period or hyphens characters", func() {
+			cases := []string{
```

Comment:
Should starting and ending with `_`,`-`,`.` be considered valid?

Reply:
From K8S documentation:

> Valid label values must be 63 characters or less and must be empty or begin and end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.

So i assume it is not considered valid.